docs/source/get_involved/governance.rst

```diff
@@ -177,13 +192,15 @@ they have been nominated, there will be a vote by the current core
 developers.
 
 Voting on appointments is one of the few activities that takes
-place on the project’s private communication channels. The vote will be
-anonymous.
+place on the project's private communication channels. The vote will be
+anonymous and should be set up by a member of the community council.
```

Comment:
```suggestion
anonymous and should be set up by a core-developer or member of the community council.
```

Reply:
come to think of it, there are just formalities. If CoCC gets reponsibility for timing the vote etc, should not a member of CoCC have the responsibility to send out all the announcements etc?

Core developers and CC members may have conflict of interest, and even if unintended this can lead to disagreements on the framing of the vote.